Mathlib/Algebra/Polynomial/Degree/Operations.lean

```diff
@@ -458,6 +458,12 @@ theorem degree_smul_le (a : R) (p : R[X]) : degree (a • p) ≤ degree p := by
 theorem natDegree_smul_le (a : R) (p : R[X]) : natDegree (a • p) ≤ natDegree p :=
   natDegree_le_natDegree (degree_smul_le a p)
 
+theorem degree_smul_of_leadingCoeff_rightRegular {k : R} (hk: k ≠ 0)
```

Comment:
When referring to a definition like `IsRightRegular`, we usually don't abbreviate it in the name, and segments should go in outer-inner order
```suggestion
theorem degree_smul_of_isRightRegular_leadingCoeff {k : R} (hk: k ≠ 0)
```